
From: David Howells <dhowells@redhat.com>

Here's a patch to add some new error codes specific to key management.

Signed-Off-By: David Howells <dhowells@redhat.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-generic/errno.h |    4 ++++
 1 files changed, 4 insertions(+)

diff -puN include/asm-generic/errno.h~add-some-key-management-specific-error-codes include/asm-generic/errno.h
--- 25/include/asm-generic/errno.h~add-some-key-management-specific-error-codes	Tue Aug 24 15:37:39 2004
+++ 25-akpm/include/asm-generic/errno.h	Tue Aug 24 15:37:39 2004
@@ -96,5 +96,9 @@
 
 #define	ENOMEDIUM	123	/* No medium found */
 #define	EMEDIUMTYPE	124	/* Wrong medium type */
+#define	ENOKEY		125	/* Required key not available */
+#define	EKEYEXPIRED	126	/* Key has expired */
+#define	EKEYREVOKED	127	/* Key has been revoked */
+#define	EKEYREJECTED	128	/* Key was rejected by service */
 
 #endif
_
